tests/environments/test_board.py

```diff
@@ -0,0 +1,42 @@
+from unittest import TestCase
+from environments.tic_tac_toe.tic_tac_toe_engine import _Board
+
+
+class TestBoard(TestCase):
+    def setUp(self):
+        size = 5
+        marks = [0, 1, 2]
+        marks_required = 3
```

Comment:
What about testing different sizes and marks_required?

Reply:
Same as below